config/settings.yml

```
@@ -4,111 +4,133 @@ col_date: "DATE"
 data_map_dict:
     dff: 
         data_ref: "DFF"
+        enabled: true
         data_source: "FRED"
 
      
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)